packages/rocket-backup-aws-infrastructure/src/utils/on-demand.ts

```diff
@@ -0,0 +1,36 @@
+import { Stack } from '@aws-cdk/core'
+import { Table } from '@aws-cdk/aws-dynamodb'
+import { BackupPlan, BackupResource, BackupPlanRule } from '@aws-cdk/aws-backup'
+import { Schedule } from '@aws-cdk/aws-events'
+import { BackupStackParams, OnDemandBackupRules } from '../backup-stack'
+
+export const applyOnDemandBackup = (stack: Stack, params: BackupStackParams, table: Table): void => {
+  // Modifiable on future versions. Other options available:
+  // - daily35DayRetention
+  // - dailyWeeklyMonthly5YearRetention
+  // - dailyWeeklyMonthly7YearRetention
+  const plan = BackupPlan.dailyMonthly1YearRetention(stack, 'BackupPlan')
+  plan.addSelection('BackupSelection', {
+    resources: [BackupResource.fromDynamoDbTable(table)],
```

Comment:
We can also do this to backup the whole framework in the future: 
`backup.BackupResource.fromTag('stage', 'prod'), // All resources that are tagged stage=prod in the region/account`